data/README.md

```diff
@@ -62,3 +62,22 @@ wp.data.query( select => {
 	};
 } )( Component );
 ```
+
+
+### `wp.data.subscribe( listener: func )`
```

Comment:
Nit: Elsewhere in this same doc you're calling this type a "function" (which seems nicer)